CMake/cdat_modules/esmf_pkg.cmake

```diff
@@ -37,11 +37,11 @@ endif()
 if(CDAT_BUILD_ESMF_ESMP)
     if("${CMAKE_Fortran_COMPILER_ID}" STREQUAL GNU)
       # GNU gfortran must be >= 4.3
-      if(${Fortran_MAJOR_VERSION} GREATER 3 AND ${Fortran_MINOR_VERSION} GREATER 2)
+      if(${Fortran_MAJOR_VERSION} GREATER 3 AND ${Fortran_MINOR_VERSION} GREATER 1)
         ## On APPLE need to test for -arch as well!
         add_cdat_package(ESMF "" "Build ESMF" ON)
       else()
-        message(FATAL_ERROR "[ERROR] gfortran must be 4.3 <= version < 5.0; you have ${Fortran_VERSION}")
+        message(FATAL_ERROR "[ERROR] gfortran must be 4.1 <= version < 5.2; you have ${Fortran_VERSION}")
```

Comment:
I didn't personally test with 5.0 or 5.1; I can definitely state that 5.2 doesn't work, though.